packages/compass-import-export/src/components/export-modal/export-modal.jsx

```diff
@@ -84,6 +84,7 @@ class ExportModal extends PureComponent {
     toggleFullCollection: PropTypes.func.isRequired,
     selectExportFileType: PropTypes.func.isRequired,
     selectExportFileName: PropTypes.func.isRequired,
+    aggregation: PropTypes.object,
```

Comment:
When mapping state to props you always want to make sure that you are passing what is required by the view, and not just passing state directly to the components. If this view only needs to know if aggregation exists or not, the logic for deriving this value should probably leave outside the component